content/millennium/r4/financial/general/charge-item.md

```diff
@@ -212,6 +212,61 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-create
+
+<%= beta_tag(action: true) %>
+
+Creates a charge event which can result in the creation of one or more ChargeItems.
+
+    POST /ChargeItem/:id/$create
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
```

Comment:
Let's remove the "unsupported fields will be ignored." as I don't think that is actually true.

Reply:
Removed https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR225